blender/arm/material/make_shader.py

```diff
@@ -97,7 +97,7 @@ def build(material: Material, mat_users: Dict[Material, List[Object]], mat_armus
             con = make_transluc.make(rp)
 
         elif rp == 'overlay':
-            con = make_overlay.make(rp)
+            con = make_mesh.make(rp, rpasses)
```

Comment:
Why did you change this here? Also, isn't make_overlay.py completely obsolete then since it's no longer called? What's the reason for the change in make_overlay.py then?

Reply:
shader needs to fill datas for the deferred shader